code/modules/projectiles/guns/projectile/boltgun.dm

```diff
@@ -12,21 +12,22 @@
 	caliber = CAL_LRIFLE
 	fire_delay = 12 // double the standart
 	damage_multiplier = 1.4
-	penetration_multiplier  = 1.5
+	penetration_multiplier = 1.5
 	recoil_buildup = 40 //same as AMR
 	handle_casings = HOLD_CASINGS
 	load_method = SINGLE_CASING|SPEEDLOADER
 	max_shells = 10
-	ammo_type = /obj/item/ammo_magazine/lrifle
+	magazine_type = /obj/item/ammo_magazine/lrifle
```

Comment:
How was this even working before?

Reply:
the variables "ammo_type" and "magazine_type" are obsolete code from the times (i guess) when Eris was a baycode. Its original function was to spawn with a specific type of "ammo" or "magazine", but currently those variables are useless, so I have taken advantage of them to integrate them into the loot rework and they have the same function that they were supposed to have originally, determine what ammunition spawns next to the weapon.